qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/AbstractRollingUpgradeTestCase.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import org.opensearch.knn.KNNRestTestCase;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+import static org.opensearch.knn.TestUtils.*;
+
+public abstract class AbstractRollingUpgradeTestCase extends KNNRestTestCase {
```

Comment:
Currently do you see differences in base functionality between abstract rolling upgrade and restart upgrade classes? Does it make sense to pull some functionality in one parent class?

Reply:
discussed offline